### DISCUSSION TITLE
Simple example of isotropic rotor blade section for CAMRAD use, part b

### DISCUSSION THREAD

**Discussion by rhig2000** [wenbinyugroup/.github#16] [VABS Related Discussions] [answered]
The expected CAMRAD offsets are XC= 0.0725671 ZC= 0.00657381. My formulas compute these for θ = 0.13525064 radians, not degrees. Are the wrong angle units stated in the VABS output file or are the expected XC, ZC values incorrect?

**Comment by wenbinyu-PU** [comment]
No, if it is in radians, should be 0.00236057. Please double check you calculation. XC, ZC are correct.

**Comment by rhig2000** [comment]
I'm calculating θ = 0.00236057 radians = 0.13525064 degrees from Eq. 52. Then, using Eq. 77:
                x2 = xT2 - xS2 = -0.07279082336
                x3 = xT3 - xS3 = -0.0032710844618
                XC = -(x2 * math.cos(θ) + x3 * math.sin(θ)) = 0.072798342
                ZC = x3 * math.cos(θ) - x2 * math.sin(θ) = -0.0030992477467

Assuming a common error is using the wrong angle units I tried using θ = 0.13525064 radians = 7.749294186 degrees in Eq. 77. resulting in XC= 0.0725671 ZC= 0.00657381. Where is my error?

**Reply by wenbinyu-PU** [reply, marked as answer]
Sorry you are right. The correct values are 0.072798342 and -0.00309925
